packages/apollos-data-prayer/src/prayer-menu-categories/data-source.js

```diff
@@ -0,0 +1,35 @@
+import { ContentItem } from '@apollosproject/data-connector-rock';
+
+import ApollosConfig from '@apollosproject/config';
+
+const { ROCK_MAPPINGS } = ApollosConfig;
+
+class PrayerMenuCategory extends ContentItem.dataSource {
+  getPrayerMenuCategories = async () => {
+    const allCategories = await this.request()
+      .filter(
+        `ContentChannelId eq ${ROCK_MAPPINGS.PRAYER_MENU_CATEGORIES_CHANNEL_ID}`
+      )
+      .get();
+    const {
+      dataSources: { Auth, Campus },
+    } = this.context;
+
+    const currentPerson = await Auth.getCurrentPerson();
+    let filteredCategories = allCategories;
+
+    // filter out campus
+    const campus = await Campus.getForPerson({ personId: currentPerson.id });
+    if (campus.name === 'Web')
+      filteredCategories = allCategories.filter(
+        (category) =>
+          category.attributeValues.dependsonCampusMembership.value === 'False'
```

Comment:
I'm definitely open to other names, I spent forever trying to come up with a good one haha. It's not camel case because it's not from Rock either: `DependsonCampusMembership` 🤷‍♂

Reply:
@redreceipt `RequiresCampusMembership`